### PR TITLE
Refactor the code along with moving towards COBRA CLI

### PR DESCRIPTION
**Pull request by kmova**
This PR will have a series of commits that will help move all the commands towards COBRA CLI 

Fixes https://github.com/openebs/openebs/issues/388
Fixes https://github.com/openebs/openebs/issues/505
Fixes https://github.com/openebs/openebs/issues/506
Fixes https://github.com/openebs/openebs/issues/513
Fixes https://github.com/openebs/openebs/issues/514

The first commit improvises the fix made by https://github.com/openebs/openebs/issues/495

Notes to the Reviewer:
I am splitting the commits based on the command that is being moved, followed by a commit that will clear the older cli/binary. For each commit, I am ensuring that the older code still works, with refactoring the functionality to a generic package. The new CLI will simply call the refactored code from a common package. 

For example, in case of moving version command, the logic of fetching the details like status and version are moved to pkg/version, pkg/client/mapiserver and orchestrators. Only the printing logic remained in the older CLI, which was easily moved to the new CLI.

### CODE REVIEW

File: pkg/client/mapiserver/status.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	addr := os.Getenv("MAPI_ADDR")
+
+	var url bytes.Buffer
+	url.WriteString(addr + "/latest/meta-data/instance-id")
+	resp, err := http.Get(url.String())
+
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
```
[the remark this answers is not in the document]
Comment (prateekpandey14) [reply pkg/client/mapiserver/status.go]: better to close body only if response non-nil , go doc https://golang.org/pkg/net/http/#Client.Do

File: pkg/client/mapiserver/status.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package mapiserver
```
[the remark this answers is not in the document]
Comment (prateekpandey14) [reply pkg/client/mapiserver/status.go]: do, we are going to have different versioning for checking the status ?

[the remark this answers is not in the document]
Comment (AmitKumarDas) [reply pkg/client/mapiserver/status.go]: i am talking overall i.e. pkg/client/mapiserver/v1

[the remark this answers is not in the document]
Comment (AmitKumarDas) [reply pkg/client/mapiserver/status.go]: why mapiserver & why not maya-apiserver ?

**Comment by kmova** [comment]
Here are the log outputs!
```
kiran@kmaya:~/go/src/github.com/openebs/maya$ ./bin/apiserver/maya-apiserver start
I1026 14:32:39.772811   17875 start.go:122] Initializing maya-apiserver...
I1026 14:32:39.772883   17875 start.go:231] Starting maya api server ...
I1026 14:32:39.772904   17875 registry.go:50] Registered 'kubernetes' as orchestration provider
I1026 14:32:39.772920   17875 registry.go:50] Registered 'nomad' as orchestration provider
I1026 14:32:39.772995   17875 start.go:160] Maya api server configuration:
I1026 14:32:39.773010   17875 start.go:162]          Log Level: INFO
I1026 14:32:39.773022   17875 start.go:162]             Region: global (DC: dc1)
I1026 14:32:39.773033   17875 start.go:162]            Version: v0.5.0-unreleased
I1026 14:32:39.773043   17875 start.go:168] 
I1026 14:32:39.773052   17875 start.go:171] Maya api server started! Log data will stream in below:
```

**Pull request merged**